stable/elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 1.3.1
+version: 1.4.1
```

Comment:
```suggestion
version: 1.4.0
```

Reply:
I updated the version.

Regarding the test, I will need to have a look at the documentation.
What do you think should be tested in this case?

Reply:
if you want to write tests, you might want to look at some examples, e.g. https://github.com/helm/charts/blob/master/stable/grafana/templates/tests/test-podsecuritypolicy.yaml

Reply:
I don't really want to add test for such a small PR, but I was respondint to @desaintmartin coment regarding https://github.com/helm/charts/tree/master/stable/elasticsearch/ci

Reply:
Oh, it is very simple, just adding a file or two in ci that acts as values.yaml testing the different configurations you are introducing (along with an empty file to also test default values).